content/docs/examples/tutorial/setup-kubernetes-cluster/index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Setup Kubernetes cluster
+overview: Set up your Kubernetes cluster for the tutorial.
+weight: 2
+---
+
+{{< boilerplate work-in-progress >}}
+
+In this module you set up a Kubernetes cluster with Istio installed and with a dedicated namespace for the tutorial.
+
+{{< warning >}}
+In case you participate in a workshop and the instructors provide a cluster for you,
+proceed to [the next module](/docs/examples/tutorial/setup-local-computer).
+{{</ warning >}}
+
+1.  Ensure you have access to a [Kubernetes cluster](https://kubernetes.io/docs/tutorials/kubernetes-basics/).
+    You can try using the [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart) or the
+    [IBM Cloud Kubernetes Service](https://cloud.ibm.com/docs/containers?topic=containers-getting-started).
+
+1.  Create an environment variable to store the name of a namespace to perform the commands of the tutorial on.
+    You can use any name, for example `tutorial`. `coolstuff` will do as well.
+
+    {{< text bash >}}
+    $ export NAMESPACE=tutorial
+    {{< /text >}}
+
+1.  Create the namespace:
+
+    {{< text bash >}}
+    $ kubectl create namespace $NAMESPACE
+    {{< /text >}}
+
+    {{< tip >}}
```

Comment:
@rcaballeromx This tip is not about setting up the cluster in general, but about allocating a different namespace for each participant. Suppose you are an instructor and you want to setup a cluster for 20 participants. You can create a single namespace and let them work together on the same namespace (overriding configurations one to another, it will not work in short). Here I am saying explicitly: allocate a namespace per each participant. This tip could be obvious, the more important sentence is: `The tutorial supports work in multiple namespaces simultaneously by multiple participants.` Another meaning is: you do not have to setup a cluster for each participant, a single cluster with multiple namespaces, a namespace per participant, will do.

It is the opposite of many Istio tasks and examples, and some other tutorials that assume that each participant has its own cluster and works in a default namespace.